srml/staking/src/phragmen.rs

```diff
@@ -214,8 +220,7 @@ pub fn elect<T: Trait + 'static, FV, FN, FS>(
 				// if the target of this vote is among the winners, otherwise let go.
 				if let Some(c) = elected_candidates.iter_mut().find(|c| c.who == e.who) {
 					e.elected = true;
-					// NOTE: for now, always divide last to avoid collapse to zero.
-					e.backing_stake = n.budget.saturating_mul(*e.load) / n.load.max(1);
+					e.backing_stake = n.budget.saturating_mul(*e.load) / (*n.load).max(1);
```

Comment:
are these parens really needed?

Reply:
```suggestion
					e.backing_stake = n.budget.saturating_mul(*e.load) / n.load.max(1);
```

Reply:
No, and this particular one does not serve any purpose frankly. Some other ones were there the emphasize the order of operations (left-associativity already works fine). All removed and replaced with comments to note the importance when needed.